stories/modal.stories.js

```diff
@@ -0,0 +1,230 @@
+              <button type="button" className="Manifold__Button Manifold__Button--Brand">
+                Save
+              </button>
+              <button type="button" className="Manifold__Button">
+                Cancel
+              </button>
+            </footer>
+          </div>
+        </div>
+      </Demo>
+      <Description>
+        <h1>Modal</h1>
+        <p>
+          Modals can be tricky to implement, especially accessible ones. Fortunately most of the
+          hassle is styling, which Mercury can take care of for you. For most of your needs, hooking
+          up functionality to a Mercury modal can be accomplished in 4 steps:
+        </p>
+        <ol>
+          <li>
+            Place your content inside a single <code>&lt;div&gt;</code>, inside your{' '}
+            <code>role="dialog"</code> element.
+          </li>
+          <li>
+            To show the modal, add <code>aria-modal="true"</code> to your <code>role="dialog"</code>{' '}
+            element.
+          </li>
+          <li>
+            To hide the modal again, remove <code>aria-modal="true"</code> (note this should happen
+            for clicks on the <strong>background overlay</strong> as well as the obvious{' '}
+            <strong>close button</strong>).
+          </li>
+          <li>
+            Add <code>overflow: hidden</code> to the <code>&lt;body&gt;</code> element while the
+            modal is visible.
+          </li>
+        </ol>
+        <p>
+          There’s an additional step that’s technically optional though highly-recommended for
+          accessibility support: pressing the <kbd>Esc</kbd> key closes the modal.
+        </p>
+        <p>
+          To see a minimal example of all this working, view the <strong>JSX</strong> tab below.
+        </p>
+      </Description>
+      <Code
+        tabs={{
+          html: `<!-- modal hidden (default) -->
+<div className="Manifold__Modal" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
+  <div>
+    <button type="button" className="Manifold__Modal__Close" aria-label="Close">✕</button>
+    <h1 id="dialog-title">Modal Title</h1>
+    <p id="dialog-desc">This is a description describing the modal.</p>
+    <footer class="Manifold__Modal__Footer">
+      <button type="submit">Save</button>
+      <button type="button">Cancel</button>
+    </footer>
+  </div>
+</div>
+
+<!-- modal visible -->
+<div aria-modal="true" className="Manifold__Modal" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
+  <div>
+    <button type="button" className="Manifold__Modal__Close" aria-label="Close">✕</button>
+    <h1 id="dialog-title">Modal Title</h1>
+    <p id="dialog-desc">This is a description describing the modal.</p>
+    <footer class="Manifold__Modal__Footer">
+      <button type="submit">Save</button>
+      <button type="button">Cancel</button>
+    </footer>
+  </div>
+</div>
+`,
+          jsx: `const HasModal = () => {
```

Comment:
Including this here will be a huge help for consistency, in my opinion. I think covering bases with easily extensible JSX like this makes sense to share with the implementer.